tests/testthat/test-glmnetUtils.R

```diff
@@ -0,0 +1,63 @@
+context("glmnetUtils")
+
+skip_if_not_installed("modeltests")
+library(modeltests)
+
+skip_if_not_installed("glmnetUtils")
+library(glmnetUtils)
+
+set.seed(27)
+
+x <- matrix(rnorm(100 * 20), 100, 20)
+y <- rnorm(100)
+g <- sample(1:4, 100, replace = TRUE)
+
+fit <- glmnet(x, y)
+fit2 <- glmnet(x, g, family = "multinomial")
+
+cv_fit <- cv.glmnet(x, y)
```

Comment:
Can you add tests that explicitly use the formula interface rather than the matrix interface?